static/app/utils/metrics/useMetricsTags.tsx

```diff
@@ -19,10 +19,19 @@ const ALLOWED_SPAN_DURATION_TAGS = [
   'span.op',
 ];
 
-export function getMetricsTagsQueryKey(organization: Organization, mri: MRI | undefined) {
-  const queryParams = {
-    metric: mri,
-  };
+export function getMetricsTagsQueryKey(
+  organization: Organization,
+  mri: MRI | undefined,
+  pageFilters: Partial<PageFilters>
+) {
+  const queryParams = pageFilters.projects?.length
+    ? {
+        metric: mri,
+        project: pageFilters.projects,
+      }
+    : {
+        metric: mri,
+      };
```

Comment:
Sending the empty array or undefined should have the same effect, right?
Asking as for other API calls we simply pass the pageFilters object without any checks.
```suggestion
  const queryParams = {
    metric: mri,
    project: pageFilters.projects,
  };
```